### PR TITLE
PedMp costumization

### PR DESCRIPTION
**Pull request by ynhhoJ**

### CODE REVIEW

**Comment by pixlcrashr** [comment]
Where is the changelog source for these changes?

File: index.d.ts [hunk 1]
```diff
@@ -361,6 +361,14 @@ interface PedMp extends EntityMp {
 	spawnPosition: Vector3Mp;
 	taskPlayAnim(animDictionary: string, animationName: string, speed: number, speedMultiplier: number, duration: number,
 		flag: number, playbackRate: number, lockX: boolean, lockY: boolean, lockZ: boolean): void;
+	setHeadOverlay(overlayID: number, index: number, opacity: number);
```
Comment (pixlcrashr) [inline index.d.ts]: Add explicit return type `void`.

**Review by pixlcrashr — CHANGES_REQUESTED** [review]

**Comment by ynhhoJ** [comment]
You can test it on RAGE MP.
For players works variant from wiki: https://wiki.rage.mp/index.php?title=Player::setHeadOverlay - ``player.setHeadOverlay(1, 4, 0.99, 1, 1);``

But if you try to use that on ped, this way doesn't work.

You need to use 3 arguments, like in native:
image
And then change color of element with ``setHeadOverlayColor``.
Idk why, but for ped costumization we need to use that way.

```js
// Doesn't work
mp.keys.bind(0x71, true, function() { // F2
    const ped = mp.peds.new(mp.game.joaat('mp_m_freemode_01'), mp.players.local.position, mp.players.local.getHeading(), mp.players.local.dimension);
    setTimeout(() => {
        ped.setHeadBlendData(1, 1, 0, 1, 1, 0, 1, 1, 0, false);
        ped.setHeadOverlay(1, 4, 0.99, 1, 1);
    }, 500);
});
```
```js
//works 
mp.keys.bind(0x71, true, function() { // F2
    const ped = mp.peds.new(mp.game.joaat('mp_m_freemode_01'), mp.players.local.position, mp.players.local.getHeading(), mp.players.local.dimension);
    setTimeout(() => {
        ped.setHeadBlendData(1, 1, 0, 1, 1, 0, 1, 1, 0, false);
        ped.setHeadOverlay(1, 4, 0.99);
        ped.setHeadOverlayColor(1, 1, 1, 1);
    }, 500);
});
```

**Comment by pixlcrashr** [comment]
Good job so far but you are missing one method `setHeadBlendData`.

**Comment by TheMysteriousVincent** [comment]
Btw. I only meant the official changelog for these API differences in version 1.0

**Review by pixlcrashr — APPROVED** [review]

**Comment by pixlcrashr** [comment]
Just to be sure: Are these changes related to the upcoming v. 1.0 or are these changes also valid for v. 0.3.7.2?

**Comment by ynhhoJ** [comment]
Like i know, It should works in 0.3.7 in the same way. But we need to do some tests to be shure 100%

**Comment by pixlcrashr** [comment]
Please test it, since we would merge directly into master.

**Review by pixlcrashr — APPROVED** [review]

**Comment by ynhhoJ** [comment]
Yep, it's also works in 0.3.7!

**Pull request merged**